contracts/insured-vesting-v1/InsuredVestingV1.sol

```diff
@@ -171,6 +172,7 @@ contract InsuredVestingV1 is Ownable {
         xctd.safeTransferFrom(msg.sender, address(this), delta);
     }
 
+    // TODO only be able to toggle if you have an allocation
```

Comment:
You think this is necessary? Why not just let them do it instead of us having to add extra logic?

Reply:
leaving the ability for anonymous (=unauthorized) users to persist data into the contract doesn't sound right to me.